hack/dockerfile/install-binaries.sh

```diff
@@ -47,10 +47,31 @@ install_proxy() {
 }
 
 install_dockercli() {
-	echo "Install docker/cli version $DOCKERCLI_COMMIT"
-	git clone "$DOCKERCLI_REPO" "$GOPATH/src/github.com/docker/cli"
-	cd "$GOPATH/src/github.com/docker/cli"
-	git checkout -q "$DOCKERCLI_COMMIT"
+	DOCKERCLI_CHANNEL=${DOCKERCLI_CHANNEL:-edge}
+	DOCKERCLI_VERSION=${DOCKERCLI_VERSION:-17.06.0-ce}
```

Comment:
sgtm other than this should be 17.06.1-ce now 😭

Reply:
I don't think so!

We want to freeze this at 17.06, we should never update this value.